### ISSUE TITLE
Remove angular-password-strength-meter

### ISSUE THREAD

**Issue by bfrgottscma** [SiLeBAT/fcl-client#619]
The library is not used by many and is quite unstable and easily replaceable by own code. E.g. the newest version 3 changed the underlying strength measurement dependency, so unsafe password are considered good (e.g. abcdefghi).

Use improved PW check (BfR PW Richtlinie (length + characterset to be used) + BSI Richtlinie (no regular change of PW)) on server.

Remove strength meter by own code (Mockup: See Mattermost --> no live check, after submitting the PW error message is shown when PW is not according to the guidelines)
Example text: Your password must be at least ... characters long, contain at least one number and one special character and have a mixture of uppercase and lowercase letters.

Also take PW reset into account.

Adapt tests to new PW Richtlinie

**Comment by bfrgottscma** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/fcl-5821aea8cbe1b0da7b31ee86/issues/SiLeBAT/fcl-client/619?planning-poker) with ZenHub @afalenski @GerberI @MicheleK17 @ruegenbfr

**Comment by ruegenbfr** [comment]
### Unchanged behaviour:
Required fields are indicated as before, 2nd password field (check) is verified as before, email is verified as before.
### Changed behaviour:
Invalid Value tests on server side only after submit (fields: first name, last name, password). 
**Please note:** The invalid value markers for these fields disappear if the field values are changed.

**Comment by afalenski** [comment]
- Entering the 'password' 'xxxxxxxx' in both password fields activates the error message telling the user what a password should contain. Good!
- The error message containg the hints for an acceptable password disappears as soon as I start typing a new password. I cannot look at the hints while creating a new password. This is not very helpful and feels as if someone makes fun of me. Please fix.
- The error message is very small and not suitable for non eagle-eyed humans. Please adjust the font size so that it is the same size as the error message for not ticking "I have read and accepted the privacy policy." (which is the same size as the name you enter, the email you give and the newsletter option).

**Comment by ruegenbfr** [comment]
- The error message containing the hints for an acceptable password disappear exactly to the issues in the reference **mattermost**. On user input change the messages regarding the input field are deleted, because the validity is uncertain. 
- The font of the error messages was not affected by the code change.  Please open another ticket for that issue.

**Comment by afalenski** [comment]
All error messages should have the same text size (choose the larger size, please).
![image.png](https://images.zenhubusercontent.com/5a854eea4b5806bc2bc70715/dbf3423e-e65e-4252-b6bb-bcaa600b4276)

**Comment by ruegenbfr** [comment]
We shouldn't compare apples with oranges, just to clarify:
On [Reference a](https://customers.mattermost.com/signup) the in my previous comment described behaviour is observable
On [Reference b](https://slack.bfr.berlin/signup_email) the from afalenski mentioned behaviour is observable

The font size needs to be addressed in another ticket, because it is not within ticket scope and not caused by the code change. Another reason for a separate ticket is in my eyes that a font size change of the error messages of the user input fields would need to be addressed in all dialogs.

**Comment by bfrgottscma** [comment]
To sum up the discussion, yes, font size of the error messages is not in the scope of the ticket but I agree that a bigger font size is better for the users to read. We agreed that @ruegenbfr is increasing the font size  of error messages in the login/registration/PW change diologues. In-app dialogues need to be checked/discussed separately and dealt with in a separate ticket.
Unfortunately, two versions of the password error message behavior exist which was not forseeable. However, I like the suggestion of @afalenski that the error message is visible throughout the editing process of the password.

**Comment by afalenski** [comment]
The error message containing the hints for an acceptable password stays visible when I start typing a new password. I can look at the hints while creating a new password. 

When in password recovery mode, the font is not red but grey. To be decided if this is going to be a new ticket...
![grafik.png](https://images.zenhubusercontent.com/5a854eea4b5806bc2bc70715/283000f5-7b0b-421a-bdce-c8bcd9aec5c5)


Done.
______
**Tested with**
Machine: BfR-Laptop

OS: Windows 10

Browsers:
- Google Chrome Version 96.0.4664.110 (Offizieller Build) (64-Bit)
- Firefox 91.7.0 (64-Bit)

**Comment by ruegenbfr** [comment]
The tabbing problem (hit TAB changes focus to error message) seems to exist only in firefox, not in Chrome, Edge or Safari. **Question:** Is this problem a release killer?

I cannot reproduce the last mentioned issue (The direct feedback of "The Password ist to short"). It is not plausible to me, because the screenshot shows the password strength meter which is not more contained in the app. I would guess this test was performed on the productive version.

**Comment by afalenski** [comment]
@ruegenbfr The problem was that on the dev server the release candidate was available and not the dev version of FCL web.
New test results:
- The error message containing the hints for an acceptable password stays visible when I start typing a new password. I can look at the hints while creating a new password.
- When in password recovery mode, the font is now also red.
- If you enter "x" as very short password into the 1st password field on the password recovery page and then hit TAB, you get directly the information that the password is too short. The same is on the register page.

 - [ ] Problem in **Firefox**, not in Chrome: When a too short password was entered and you can see the red password information, position the cursor in the 1st password field: If you hit TAB, then you do not get to the 2nd password field but you land "in" the information box with the red font. This textbox should not be reachable by hitting TAB. You should get directly from the 1st to the 2nd password field as it was before. To be decided if this is going to be a new ticket...
![grafik.png](https://images.zenhubusercontent.com/5a854eea4b5806bc2bc70715/568544d2-c556-42c7-8eb9-c6a7f7a82aec)
______
**Tested with**
Machine: 
- BfR-Laptop

OS: Windows 10

Browsers:
- Firefox 91.10.0esr (64-Bit)
- Chrome 96.0.4664.110 (64-Bit)

**Comment by bfrgottscma** [comment]
For me it is not a release stopper, when hitting TAB leads you to the information box. However, a new ticket should be created which fixes this problem but it is not a problem if this is released in a later release.

**Comment by ruegenbfr** [comment]
Tab problem fixed.
The hint cannot get anymore the focus by using tab.

**Comment by afalenski** [comment]
- [ ] The error message containing the hints for an acceptable password stays visible when I start typing a new password. I can look at the hints while creating a new password.
- [ ] When in password recovery mode, the font is now also red.
- [ ] If you enter "x" as very short password into the 1st password field on the password recovery page and then hit TAB, you get directly the information that the password is too short. The same is on the register page.
- [ ] The Problem in Firefox that you land "in" the information box with the red font is fixed and does not occur in Chrome, either.

Done.
______
**Tested with**
Machine: BfR-Laptop

OS: Windows 10

Browsers:
- Firefox 91.10.0esr (64-Bit)
- Google Chrome Version 102.0.5005.115 (Offizieller Build) (64-Bit)
